mod/profile_photo.php

```diff
@@ -158,78 +155,77 @@ function profile_photo_post(App $a) {
 	$imagedata = @file_get_contents($src);
 	$ph = new Image($imagedata, $filetype);
 
-	if (! $ph->isValid()) {
+	if (!$ph->isValid()) {
 		notice(L10n::t('Unable to process image.') . EOL);
 		@unlink($src);
 		return;
 	}
 
 	$ph->orient($src);
 	@unlink($src);
-	return profile_photo_crop_ui_head($a, $ph);
```

Comment:
I guess we never use the return value?

Reply:
Worst than that, it returns an array which the `module_post()` function isn't supposed to.